posthog/temporal/tests/batch_exports/test_s3_batch_export_workflow.py

```diff
@@ -2363,7 +2363,7 @@ def _create_test_record_batch() -> pa.RecordBatch:
                 pa.array([json.dumps({"set1": "value1"})]),  # set
                 pa.array([json.dumps({"set_once1": "value1"})]),  # set_once
             ]
-            return pa.RecordBatch.from_arrays(arrays, schema=schema)
+            return pa.RecordBatch.from_arrays(arrays, schema=schema)  # type: ignore
```

Comment:
Couldn't we add a type hint for arrays, like: `arrays: collections.abc.Collection[pa.Array[typing.Any]] = [`